guide/guide.html

```diff
@@ -735,9 +735,9 @@ <h3>Prototypes</h3>
 
 
 
-                    In Io, everything is an object (including the locals storage of a block and the namespace itself) and all actions are messages (including assignment). Objects are composed of a list of key/value pairs called slots, and an internal list of objects from which it inherits called protos. A slot's key is a symbol (a unique immutable sequence) and its value can be any type of object.
+                    In Io, everything is an object (including the locals storage of a block and the namespace itself) and all actions are messages (including assignment). Objects are composed of a list of key/value pairs called slots, and an internal list of objects from which they inherit called protos. A slot's key is a symbol (a unique immutable sequence) and its value can be any type of object.
```

Comment:
I think that `it inherits` becomes `they inherit` because `they` refers to `Objects` and must agree in number.